soundata/version.py

```diff
@@ -2,4 +2,4 @@
 """Version info"""
 
 short_version = "0.1"
-version = "0.1.0-rc14"
+version = "0.1.0"
```

Comment:
This seems like a big PR. Let's first merge this one in as rc15m and then have a separate PR just for the official update to v0.1.0 and the release